applications/nrf_desktop/doc/bas.rst

```diff
@@ -23,7 +23,7 @@ Configuration
 *************
 
 The module is enabled with the :ref:`CONFIG_DESKTOP_BAS_ENABLE <config_desktop_app_options>` option.
-The option is selected by :ref:`CONFIG_DESKTOP_HID_PERIPHERAL <config_desktop_app_options>` -- Battery Service is required for the HID peripheral device.
+The option is implied by :ref:`CONFIG_DESKTOP_BT_PERIPHERAL <config_desktop_app_options>` -- Battery Service is required for the HID peripheral device.
```

Comment:
Could we merge it with the previous sentence to say: 
The module is enabled with the :ref:`CONFIG_DESKTOP_BAS_ENABLE <config_desktop_app_options>` option, that is implied by the :ref:`CONFIG_DESKTOP_BT_PERIPHERAL <config_desktop_app_options>` option.
(from the new line) The Battery Service is required for the HID peripheral device.  
?